docs/intro/operators.md

```diff
@@ -198,64 +154,83 @@ In this given example, the `globalResponseTimeout` is set to `5000` milliseconds
 It's crucial to set an appropriate response timeout, especially in production environments. This not only optimizes resource utilization but also acts as a security measure against potential denial-of-service attacks where adversaries might run complex queries to exhaust server resources.
 :::
 
+
+## @upstream
+
+The `upstream` directive allows you to control various aspects of the upstream server connection. This includes settings like connection timeouts, keep-alive intervals, and more. If not specified, default values are used.
+
+```graphql showLineNumbers
+schema @upstream(...[UpstreamSetting]...){
+    query: Query
+    mutation: Mutation
+}
+```
+
+Below is a concise description of each attribute within the `upstream` directive:
+
+- `poolIdleTimeout`: The time in seconds that the connection pool will wait before closing idle connections.
+- `poolMaxIdlePerHost`: The maximum number of idle connections that will be maintained per host.
+- `keepAliveInterval`: The time in seconds between each keep-alive message sent to maintain the connection.
+- `keepAliveTimeout`: The time in seconds that the connection will wait for a keep-alive message before closing.
+- `keepAliveWhileIdle`: A boolean value that determines whether keep-alive messages should be sent while the connection is idle.
+- `proxy`: An object that specifies the proxy server's URL. more information about the proxy server is provided below.
+- `connectTimeout`: The time in seconds that the connection will wait for a response before timing out.
+- `timeout`: The maximum time in seconds that the connection will wait for a response.
+- `tcpKeepAlive`: The time in seconds between each TCP keep-alive message sent to maintain the connection.
+- `userAgent`: The User-Agent header value to be used in HTTP requests.
+- `allowedHeaders`: An array of HTTP headers that are allowed to be forwarded to upstream services. More information about the allowed header of the upstream server is provided below.
+- `baseURL`: The base URL of the upstream server. More information about the base URL of the upstream server is provided below.
+- `enableHttpCache`: when activated, directs Tailcall to utilize HTTP caching mechanisms. These mechanisms, in accordance with the [HTTP Caching RFC](https://tools.ietf.org/html/rfc7234), are designed to improve performance by reducing unnecessary data fetches. If left unspecified, this feature defaults to `false`.
+- `batch`: An object that specifies the batch settings, including `maxSize` (the maximum size of the batch), `delay` (the delay in milliseconds between each batch), and `headers` (an array of HTTP headers to be included in the batch).
```

Comment:
Instead of bullets I think we should keep the formatting consistent with other such as proxy and baseURL etc. Meaning use `####` instead of `-`

Reply:
@tusharmath 
done. please review!